src/hotspot/share/utilities/growableArray.hpp

```diff
@@ -720,15 +723,31 @@ class GrowableArray : public GrowableArrayWithAllocator<E, GrowableArray<E> > {
   }
 
 public:
-  GrowableArray(int initial_capacity = 2, MEMFLAGS memflags = mtNone) :
+  explicit GrowableArray(int initial_capacity = 2) :
+      GrowableArrayWithAllocator<E, GrowableArray<E> >(
+          allocate(initial_capacity),
+          initial_capacity),
+      _metadata() {
+    init_checks();
+  }
+
+  explicit GrowableArray(MEMFLAGS memflags, int initial_capacity = 2) :
```

Comment:
Am I right in thinking that 99% of this MR was only necessary because you have to switch the argument order because you no longer have/want a default for memflags?